File: week-02/day-03/element_finder.py
# Check if the list contains "7" if it contains print "Hoorray" otherwise print "Noooooo"

numbers = [1, 2, 3, 4, 5, 6, 8];
contains_seven = False

for current_number in numbers:
    if current_number == 7:
        contains_seven = True
if contains_seven:
    print("Hoooray")
else:
    print("No")

File: week-07/day-02/element-finder.js
'use strict';

// Check if the array contains "7" if it contains print "Hoorray" otherwise print "Noooooo"

var numbers = [1, 2, 3, 4, 5, 7, 8];
var i = 0;
var x = false;

for (i = 0; i < numbers.length; i++) {
    if (numbers[i] === 7) {
        x = true;
    }
}
if (x) {
    console.log("Huray");
} else {
    console.log("Nooo");
}
